src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,104 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private List<Entry<K, V>>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new ArrayList[INITIAL_CAPACITY];
```

Comment:
1. please, rewrite without using of ready collections
2. no need to use `this` keyword
```suggestion
    private Entry<K, V>[] table;
    private int size;

    public MyHashMap() {
        table = new Entry[INITIAL_CAPACITY];
```